cartography/intel/gcp/databases.py

```diff
@@ -0,0 +1,864 @@
+    session.write_transaction(_load_bigtable_instances_tx, data_list,project_id, update_tag)
+    
+@timeit
+def _load_bigtable_instances_tx(tx: neo4j.Transaction,bigtable_instances: List[Dict],project_id: str,gcp_update_tag: int) -> None:
+    """
+        :type neo4j_transaction: Neo4j transaction object
+        :param neo4j transaction: The Neo4j transaction object
+
+        :type instances_resp: List
+        :param instances_resp: A list of Bigtable Instances
+
+        :type project_id: str
+        :param project_id: Current Google Project Id
+
+        :type gcp_update_tag: timestamp
+        :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+    """
+    ingest_bigtable_instances = """
+    UNWIND {bigtable_instances} as instance
+    MERGE (i:GCPBigtableInstance{id:{instance.id}})
+    ON CREATE SET
+        i.firstseen = timestamp()
+    SET
+        i.name = instance.name,
+        i.displayName = instance.displayName,
+        i.state = instance.state,
+        i.type = instance.type,
+        i.createTime = instance.createTime
+    WITH instance, i
+    MATCH (owner:GCPProject{id:{ProjectId}})-[r1:Resource]->(i:GCPBigtableInstance{id:{instance.id}})
+    MERGE (owner)-[r:RESOURCE]->(i)
+    ON CREATE SET
+        r.firstseen = timestamp(),
+        r.lastupdated = {gcp_update_tag}
+    """
+    tx.run(
+        ingest_bigtable_instances,
+        bigtable_instances = bigtable_instances,
+        ProjectId=project_id,
+        gcp_update_tag=gcp_update_tag,
+    )
+
+@timeit
+def load_bigtable_clusters(session: neo4j.Session, data_list: List[Dict[str, Optional[str]]],project_id: str,update_tag: int) -> None:
+    session.write_transaction(_load_bigtable_clusters_tx, data_list,project_id, update_tag)
+    
+@timeit
+def _load_bigtable_clusters_tx(tx: neo4j.Transaction,bigtable_clusters: List[Dict],project_id: str,gcp_update_tag: int) -> None:
+    """
+        :type neo4j_transaction: Neo4j transaction object
+        :param neo4j transaction: The Neo4j transaction object
+
+        :type clusters_resp: List
+        :param clusters_resp: A list of Bigtable Instance Clusters
+
+        :type project_id: str
+        :param project_id: Current Google Project Id
+
+        :type gcp_update_tag: timestamp
+        :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+    """
+    ingest_bigtable_clusters = """
+    UNWIND {bigtable_clusters} as cluster
+    MERGE (c:GCPBigtableCluster{id:{cluster.id}})
+    ON CREATE SET
+        c.firstseen = timestamp()
+    SET
+        c.name = cluster.name,
+        c.location = cluster.name,
+        c.state = cluster.state,
+        c.serveNodes = cluster.serveNodes,
+        c.defaultStorageType = cluster.defaultStorageType
+    WITH cluster,c
+    MATCH (owner:GCPProject{id:{ProjectId}})-[r1:Resource]->(i:GCPBigtableInstance{id:{instance.id}})-[r2:Resource]->(c:GCPBigtableCluster{id:{cluster.id}})
```

Comment:
Same as above. We do not need to match from project all the way to bigtablecluster. we just need to build relationship between current node and the nearest node.
for ex: in this case, we just need to build relationship between cluster and instance. Instance is already related to project via `_load_bigtable_instances_tx`